samcli/commands/build/utils.py

```diff
@@ -0,0 +1,110 @@
+"""
+Utilities for sam build command
+"""
+import pathlib
+from enum import Enum
+from typing import List
+
+import click
+
+from samcli.lib.build.workflow_config import CONFIG, get_workflow_config
+from samcli.lib.providers.provider import ResourcesToBuildCollector
+
+
+class MountMode(Enum):
+    """
+    Enums that represent mount mode used when build lambda functions/layers inside container
+    """
+
+    READ = "READ"
+    WRITE = "WRITE"
+
+    @classmethod
+    def values(cls) -> List[str]:
+        """
+        A getter to retrieve the accepted value list for mount mode
+
+        Returns: List[str]
+            The accepted mount mode list
+        """
+        return [e.value for e in cls]
```

Comment:
If we can use `_members_` for click option, then we might remove this function here as well